redbeams/src/main/java/com/sequenceiq/redbeams/converter/stack/AllocateDatabaseServerV4RequestToDBStackConverter.java

```diff
@@ -58,6 +57,8 @@
 
     private static final DBStackStatus NEW_STATUS = new DBStackStatus();
 
+    private static final List<CloudPlatform> SUPPORTED_CLOUD_PLATFORMS = List.of(CloudPlatform.AWS);
```

Comment:
Can you add Azure to this list as well? I'll be checking in Azure support shortly.

Reply:
Yepp, I added.